classroom/snippets/snippets.py

```diff
@@ -13,13 +13,110 @@
 # limitations under the License.
 
 from __future__ import print_function
+import json
 from googleapiclient import errors
 
 
 class ClassroomSnippets(object):
     def __init__(self, service):
         self.service = service
 
+    def create_course(self):
+        """
+        Creates a 10th Grade Biology course.
+        """
+        service = self.service
+        # [START classroom_create_course]
+        course = {
+            'name': '10th Grade Biology',
+            'section': 'Period 2',
+            'descriptionHeading': 'Welcome to 10th Grade Biology',
+            'description': """We'll be learning about about the
+                         structure of living creatures from a
+                         combination of textbooks, guest lectures,
+                         and lab work. Expect to be excited!""",
+            'room': '301',
+            'ownerId': 'me',
+            'courseState': 'PROVISIONED'
+        }
+        course = service.courses().create(body=course).execute()
+        print('Course created:', course.get('name'), course.get('id'))
```

Comment:
This is out of scope for this review but let's make sure any code snippets you're not replacing here are Python 3:
https://developers.google.com/classroom/guides/manage-courses